mep/books/admin.py

```diff
@@ -455,41 +447,29 @@ def before_import_row(self, row, **kwargs):
         )
 
 
-def unique_list(l):
-    l2 = []
-    for x in l:
-        if x not in set(l2):
-            l2.append(x)
-    return l2
+class NamedListWidget(Widget):
+    sep = ";"
 
+    @classmethod
+    def render(cls, queryset):
+        return cls.sep.join(child.name for child in queryset.all())
```

Comment:
For some reason I can't get this to run when inserting it as `widget=NamedListWidget` into the `Field(...)` code below, so for now just invoking it in the dehydration.